### PR TITLE
Servers definition

### PR DESCRIPTION
**Pull request by Vlczech**
Allows to define servers with url, description and variables as is defined here: https://swagger.io/docs/specification/api-host-and-base-path/

### CODE REVIEW

File: spectree/spec.py [hunk 1]
```diff
@@ -264,6 +264,16 @@ def _generate_spec(self):
...
 
+        if self.config.SERVERS:
+            spec["servers"] = [
+                {
+                    "url": server.url,
+                    "description": server.description,
+                    "variables": server.variables,
+                }
+                for server in self.config.SERVERS
+            ]
```
Comment (kemingy) [inline spectree/spec.py:275]: Is it better to use `[server.dict() for server in self.config.SERVERS]`?

**Review by kemingy — COMMENTED** [review]
Thanks for your contribution.

**Review by kemingy — APPROVED** [review]
LGTM

File: spectree/spec.py (hunk 1 above)
Comment (Vlczech) [reply spectree/spec.py]: It seems good - I used vars or something like this previously, but the test failed due to different key positions. But this is OK :-)

**Pull request merged**